content/blog/2022-06-29-june-community-gems.md

```diff
@@ -0,0 +1,215 @@
+  working with the DVC cache, DVC data and remotes, using DVC programmatically,
+  and more.
+picture: 2022-06-29/june-community-gems.png
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/may-22-community-gems/1184
+tags:
+  - DVC Remotes
+  - Pipelines
+  - CML
+  - GCP
+  - Community
+---
+
+## [Is there something like `git add-commit -m "some message"` for DVC?](https://discord.com/channels/485586884165107732/563406153334128681/981498675689828362)
+
+Thanks for the question @Ramnath T!
+
+While there isn't a single command to handle this, you can get similar behavior
+with DVC with an alias, like the following:
+
+```dvc
+$ alias dvc-add-commit='dvc add . & dvc commit'
+```
+
+When you run `dvc add <file name or folder name>`, the data will be added to
+your local cache and no commit is needed. This is how we make DVC aware of any
+new data we want versioned. If you already have data tracked by DVC, the
+`dvc commit` command adds all the changes to those files or directories without
+having to name each target. You'll still need to remember to commit any other
+changes you've made to Git as well.
+
+When you run `dvc add`, a file hash will be calculated, the file content will be
+moved to the cache, and a `.dvc` file will be created to start tracking the
+added data. If you're working with remotes using the `--to-remote` option, you
+can skip the local cache entirely and move the file contents directly to your
+remote storage.
+
+## [How can I connect Iterative Studio to a remote repo on a private network, like a GitLab server?](https://discord.com/channels/485586884165107732/563406153334128681/981543978644172830)
+
+Good question about [Iterative Studio](https://studio.iterative.ai/) from
+@LilDataScientist!
+
+This is something that our users asked quite a bit, so we wrote up a whole guide
+about
+[custom GitLab server connections](https://dvc.org/doc/studio/user-guide/connect-custom-gitlab-server).
+It's a quick walkthrough of how to set up the permissions you'll need and
+connecting your team to Studio.
+
+You can find lots of great guides and explanations about everything Studio in
+the [User Guide](https://dvc.org/doc/studio/user-guide) section of the docs!
+
+## [How does `dvc get-url` interact with the cache compared to `dvc import-url`?](https://discord.com/channels/485586884165107732/563406153334128681/981862313076346920)
+
+This is an awesome question from @Gema Parreno!
+
+When you run `dvc get-url`, it downloads the file/directory to your local file
+system. It's _not_ tracking the downloaded data with a `.dvc` file. It's just
+pulling that data from some source to your file system. If you want to download
+a file or directory without needing a DVC project, you can use the `dvc get-url`
+command.
+
+On the other hand, when you run `dvc import-url`, the local `cache` folder
+inside of `.dvc` will be updated. This is similar to running `dvc get-url` and
+`dvc add` together.
```

Comment:
Except that `dvc import-url` also saves a link to the original file/directory location so that if it changes, you can download the updated file/directory.